.github/workflows/tekton_task_tests.yaml

```diff
@@ -22,8 +22,26 @@ jobs:
           dir_names: "true"
           dir_names_max_depth: "3"
       - name: Show changed dirs
+        id: show-changed-dirs
+        shell: bash
         run: |
           echo ${{ steps.changed-dirs.outputs.all_changed_files }}
+          trustedArtifactsBasedTasks=()
+          for dir in ${{ steps.changed-dirs.outputs.all_changed_files }}
+          do
+            echo "$dir"
+            baseDir=$(basename "${dir}")
```

Comment:
Maybe call this `taskName` instead of `baseDir` - makes it more obvious what it is.

Reply:
fixed